tomaty/tomaty.py

```diff
@@ -98,6 +104,17 @@ def __init__(self):
         self.notebook.append_page(
             child=self.statsPage, tab_label=Gtk.Label(label="stats"))
 
+
+        self.connect('delete_event', self.destory)
+
+    def destory(self, widget=None, *data):
+        """
+            save data before exiting
+        """
```

Comment:
again, i have not finished this myself with the rest of the code, but can you please add to your docstring the args passed in the format of:
```python
"""
one sentence descrip (ya good here)

:param widget: blah
:param *data: blah
:return identifier: you don't return anything but just putting here as example

long description if wanted/need it
"""
```

thank you!!!!